### DISCUSSION TITLE
Where do I give feedback on the new docs? (+feedback)

### DISCUSSION THREAD

**Discussion by stanlrt** [solidjs/solid#2187] [General]
The TypeForm the banner links to is now closed.

Anyway I'll put the feedback here: 
The loss of the list showing all the functions (createXXX) provided by Solid is annoying for new adopters. 
I'm new to Solid; I don't want to look for a specific usecase like the current table of contents encourages. Instead, I want an overview of everything Solid offers.

**Comment by atk** [comment]
You can switch between Learn and Reference at the top of the navigation. Maybe we should add another button beneath "Get started" to make it easier to find.

**Reply by stanlrt** [reply]
I completely missed that, thanks
